packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -39,36 +70,67 @@ export const AWS_CDK_METADATA = new Set([
  * @see https://docs.aws.amazon.com/general/latest/gr/s3.html#s3_region
  */
 export const ROUTE_53_BUCKET_WEBSITE_ZONE_IDS: { [region: string]: string } = {
+  // Africa (Cape Town)
```

Comment:
I'm sure this was all a great lot of effort to put all the comments in here... but why? 

In all honestly, I don't see the value this adds.

Reply:
When looked up the fact-tables file, it looked like some fields were setup alphabetically while for some contributions from open community added comments specifying the regions. So, wanted to make it consistent throughout which will help when we want to add a new region or find the missing values for any region when looked up.

Reply:
Alphabetization seems definitely like a good idea. 👍🏻

Reply:
yes, added comments for regions because `CLOUDWATCH_LAMBDA_INSIGHTS_ARNS` section in the fact-tables file has changes specifying regions so added those to make it look clean. But, will get those out as discussed.